src/main/java/io/github/prospector/modmenu/mixin/MixinTitleScreen.java

```diff
@@ -18,14 +20,26 @@ public class MixinTitleScreen extends Screen {
 	@Inject(at = @At("RETURN"), method = "init")
 	public void drawMenuButton(CallbackInfo info) {
 		ButtonWidget texturePackButton = minecraft.isApplet ? (ButtonWidget) this.buttons.get(this.buttons.size() - 2) : (ButtonWidget) this.buttons.get(2);
-		texturePackButton.text = "Texture Packs";
-		int newWidth = ((MixinGuiButton) texturePackButton).getWidth() / 2 - 1;
-		((MixinGuiButton) texturePackButton).setWidth(newWidth);
-		this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 + 2, texturePackButton.y, newWidth, 20,  "Mods (" + ModMenu.getFormattedModCount() + " loaded)"));
+
+		if (FabricLoader.getInstance().isModLoaded("legacytranslations")) {
+			TranslationStorage var2 = TranslationStorage.getInstance();
+			texturePackButton.text = var2.get("menu.texturepacks");
+			int newWidth = ((MixinGuiButton) texturePackButton).getWidth() / 2 - 1;
+			((MixinGuiButton) texturePackButton).setWidth(newWidth);
+			this.buttons.add(new ModMenuButtonWidget(100, this.width / 2 + 2, texturePackButton.y, newWidth, 20,  var2.get("menu.mods") + " (" + ModMenu.getFormattedModCount() + ")"));
```

Comment:
Same here

Reply:
Added back in, and upgraded LegacyTranslations to be able to handle button text overflow